vunit/sim_if/vsim_simulator_mixin.py

```diff
@@ -257,6 +257,14 @@ def _create_gui_script(self, common_file_name, config):
         """
         tcl = f'source "{fix_path(common_file_name)!s}"\n'
         tcl += self._create_user_init_function(config)
+        tcl += 'add_menu "" VUnit\n'
+        tcl += 'add_menuitem "" VUnit "vunit user init" vunit_user_init\n'
```

Comment:
To follow the style of other menu items it should be "VUnit User Init" instead of "vunit user init"

Reply:
Fixed